src/main/java/nl/dtls/fairdatapoint/service/jwt/JwtService.java

```diff
@@ -55,6 +56,7 @@
     @Autowired
     private UserRepository userRepository;
 
+    @Lazy
     @Autowired
     private AuthenticationManager authenticationManager;
```

Comment:
Is there any fundamental change how spring handles this in this version?

Reply:
I am not totally sure (based on reading the CHANGELOG) what caused this, but it detected dependency loop with `SecurityConfig` and `JwtTokenFilter`... This loop was always there but caused no issues. Probably they changed something in initialization order of filters/configs 🤷🏻‍♂️